src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+import { TodoWithoutUser } from './TodoWithoutUser';
+import { User } from './User';
+
+export interface Todo extends TodoWithoutUser {
+  user: User | null;
```

Comment:
Consider using a more descriptive name for the property instead of `user`. For example, `assignedUser` or `owner`. 🤔